### PR TITLE
Keras dev

### PR DESCRIPTION
**Pull request by Onoyiza**
Hi @pgleeson. 
I am currently working on updating the readme while trying to work on creating a convolution example that works. I want to test that the batch normalization and dropout nodes currently work well, so I might update the Keras model on the MNIST data to include those layers. 
I opened this draft pull request so that the changes I make can be tracked. You can merge these changes at any time.
Thank you.

### CODE REVIEW

**Comment by pgleeson** [comment]
@Onoyiza I'll just merge in the latest version of the readme and check it through. Feel free to update with more info if you like
